docs/about-ensembl/the-ensembl-project/cite.md

```diff
@@ -0,0 +1,24 @@
+---
+slug: cite
+title: Citing Ensembl
+description:
+status: draft
+---
+
+# Citing Ensembl
+
+If you've used Ensembl in your work, please cite the appropriate overview article:
+
+## For vertebrate genomes
+
+*Ensembl 2020*
+_Nucleic Acids Res._ 2021, vol. 49(1):884–891
+[doi:10.1093/nar/gkaa942](https://doi.org/10.1093/nar/gkaa942)
```

Comment:
Do you expect this to be rendered on three separate lines? If so, then I believe the first two lines should end in `\`.

Reply:
Oh, OK - will fix that (and the one below)

Reply:
It's confusing, because in Unix the backslash means "continue on same line"!